giraffe/src/types/index.ts

```diff
@@ -916,3 +924,44 @@ export interface AnnotationMark {
   pin: AnnotationPinType
   id?: string
 }
+
+export enum ComponentColor {
```

Comment:
@TCL735 is this too much of Clockface leaking into giraffe?

Reply:
No, but they don't belong here either. I will make another comment.